server/src/TPlayer.cpp

```diff
@@ -4165,45 +4177,54 @@ bool TPlayer::msgPLI_SENDTEXT(CString& pPacket)
 					// Irc players start at 16k
 					sendPacket(CString() >> (char)PLO_ADDPLAYER >> (short)(16000 + 0) >> (char)channelAccount.length() << channelAccount >> (char)PLPROP_NICKNAME >> (char)channelNick.length() << channelNick >> (char)81 >> (char)3);
```

Comment:
I think playerId 16000 is reserved, just like playerid below 2 or something was reserved. I think when playerid 16000 is added it counts as "onServerlistConnected". Then IRC channels and external players are on >16000

Reply:
Because, the v5 client won't send GraalEngine,irc,login,- until playerId 16000 is added.